src/modules/phoneInput.js

```diff
@@ -154,7 +176,7 @@ class CountryDropdown extends React.PureComponent {
         return (
             <Dropdown
                 clearable={false}
-                disabled={disabled}
+                disable={disabled}
```

Comment:
Thanks for fixing that.  _Someday_ I'd like to fix `<Dropdown />` so it really is `disabled` (for consistency), but that's a larger effort.  See #25.

Reply:
I agree.

Reply:
We probably could change them all to `disable` and have a `disabed` alias for the time being.

Reply:
Do you preferred `disable` or `disabled`?  I prefer `disabled` myself.

Reply:
`disable`

Reply:
Although, my original thought behind `disabled` is based on w3 specs. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#htmlattrdefdisabled

Reply:
Yes exactly ... I feel `disabled` is a bit more conventional.

Reply:
And I think for _most_ of React CM UI, we've used `disabled`.  I think `<Dropdown />` is an outlier (and that there's maybe one or two others also using `disable` but most use `disabled`).